helix-rest/src/main/java/org/apache/helix/rest/metadatastore/accessor/ZkRoutingDataReader.java

```diff
@@ -90,10 +99,12 @@ public ZkRoutingDataReader(String namespace, String zkAddress,
     for (String realmAddress : allRealmAddresses) {
```

Comment:
Null check on allRealmAddress just in case?

Reply:
I'll add it, but is that case actually possible?